servicetalk-loadbalancer/src/main/java/io/servicetalk/loadbalancer/LoadBalancingPolicies.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.loadbalancer;
+
+/**
+ * A factory to create default {@link LoadBalancingPolicy LoadBalancingPolicies}
+ */
+final class LoadBalancingPolicies {
```

Comment:
The value of this is perhaps limited right now.

Reply:
I'll remove this class for now since it's just shortcuts. We can always make shortcuts later.